src/api/helpers/mapSicCodes.ts

```diff
@@ -0,0 +1,54 @@
+interface CompanyResponse {
+  id: string | null;
+  applicationId: string | null;
+}
+
+interface ConnectId {
+  id: string;
+}
+
+interface ConnectObj {
+  connect: ConnectId;
+}
+
+interface SicCodes {
+  code: string;
+  exporterCompany: ConnectObj;
+  application: ConnectObj;
+}
```

Comment:
Can we move these into existing `types.ts` file please

Reply:
done